CHANGELOG.md

```diff
@@ -1,5 +1,31 @@
 # Mattermost Mobile Apps Changelog
 
+## v1.12.0 Release
+- Release Date: September 16, 2018
+- Server Versions Supported: Server v4.10+ is required, Self-Signed SSL Certificates are not supported
+
+### Highlights
+
+#### Search Date Filters
+- Search for messages before, on, or after a specified date.
+
+### Improvements
+- Added notification support for Android O and P.
+- Added ability to open channels via the new message indicator.
```

Comment:
What is this?

Reply:
I realized it was something related to the Scrolling Overhaul feature so I removed it from here.